dinstaller-cli/src/main.rs

```diff
@@ -64,7 +64,18 @@ async fn show_progress(client: &ManagerClient<'_>) {
     let mut progress = client.progress().await.unwrap();
     eprintln!("Showing progress with max steps {:?}", progress.max_steps);
     let pb = ProgressBar::new(progress.max_steps.into());
+    let questions = QuestionsClient::new(dinstaller_lib::connection().await.unwrap()).await.unwrap();
```

Comment:
Once #38 is merged, we can adapt this code not to call `unwrap()` so often.

Reply:
I thought it would be a nice simple task but meanwhile the progress reporting has been upgraded too much :rofl: